packages/ui/src/pages/Profile/index.module.scss

```diff
@@ -0,0 +1,40 @@
+@use '@/scss/underscore' as _;
+
+.container {
+  @include _.flex-column;
```

Comment:
Nit:
The `_.flex-column` has a default `justify-content: center` prop; this may cause unexpected overflow behavior on a small-size window. (the parent content will cover the content and can not scroll to make the content fully show up)

Reply:
Thanks